projects/common/lib/components/date/date.component.html

```diff
@@ -37,6 +39,7 @@
            (blur)="onBlurYear($event.target.value)"
            commonDateFieldFormat
            [disabled]='disabled'
+           [required]="isRequired"
```

Comment:
@kreed0422 This is an example of a component that is already using a input for `required`

I have not found a way to get the required attribute onto the actual input automatically, even with the use of things like ControlValueAccessor. When I add just add `required` to the custom form control it applies to the tags of the form control but not the actual input element within, which isn't what Ernesto is looking for.